docs/architecture/adr/0017-watchOS-use-swift.md

```diff
@@ -0,0 +1,57 @@
+---
+adr: "0017"
+status: Done
+date: 2022-12-30
+tags: [clients, watchOS]
```

Comment:
`clients` denote the repository, since this is the mobile application we should use the mobile tag.

```suggestion
tags: [mobile, watchOS]
```